azure-simple/packer/ansible-is/dev/inventory

```diff
@@ -0,0 +1,3 @@
+[is]
+is_1 ansible_host=localhost ansible_user=centos ansible_connection=local
+
```

Comment:
@rasanjaya85 shall we remove this unnecessary EOF line?

Reply:
Done